content/docs/develop/reference/troubleshooting/index.md

```diff
@@ -37,6 +41,33 @@ In this section, instructions are summarized that help to trouble shoot known is
     ```
     kubectl delete pods --all -n knative-eventing
     ```
+</p></details>
+
+## Control Servic is not available
+
+### CLI command was not executed correctly
+<details><summary>Expand instructions</summary>
+<p>
+
+**Investigation:**
+
+The control service is not available at the time when a command was sent by the keptn CLI. 
+The resulting response message will look similar to this:
+
+    ```console
+    keptn onboard service --project=sockshop --values=values_carts.yaml
+    
+    Starting to onboard service
+    Onboard service was unsuccessful
+    Error: Post https://control.keptn.1xx.xxx.xx.xx.xip.io/service: dial tcp: lookup control.keptn.1xx.xxx.xx.xx.xip.io: no such host
+    ``` 
+
+**Reason:** 
+
+Probably, the cluster is overloaded
```

Comment:
That is somewhat vague - can we offer more guidance here to verify that the cluster is overloaded?

Reply:
Unfortunately, we do not know the exact reasons.